### ISSUE TITLE
MySQL Database migrator fails on creating a database with dashes in the name

### ISSUE THREAD

**Issue by baschtl** [hanami/model#381]
Running this

```ruby
Hanami::Model::Migrator::MySQLAdapter.create
```

fails with 

```
Sequel::DatabaseError: Mysql2::Error: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'db-name' at line 1
```

when the database name includes dashes, e.g., `db-name`.

I guess the issue can be found in https://github.com/hanami/model/blob/master/lib/hanami/model/migrator/mysql_adapter.rb#L22. And probably

```
%(CREATE DATABASE `#{database}`;)
```

solves the issue.

**Comment by jodosha** [comment]
@baschtl Hi and thank you for reporting this. I created a PR with the fix #383. Can you please check it? Thanks.

**Comment by baschtl** [comment]
@jodosha Thanks for addressing this. Looks good! :)
